homeassistant/components/ecobee/notify.py

```diff
@@ -39,3 +60,50 @@ def send_message(self, message="", **kwargs):
         for target in targets:
             thermostat_index = int(target)
             self.ecobee.send_message(thermostat_index, message)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the ecobee thermostat."""
+
+    data: EcobeeData = hass.data[DOMAIN]
+    entities = []
+
+    for index in range(len(data.ecobee.thermostats)):
+        thermostat = data.ecobee.get_thermostat(index)
+        entities.append(EcobeeNotifyEntity(data, index, thermostat))
+
+    async_add_entities(entities, True)
+
+
+class EcobeeNotifyEntity(NotifyEntity):
+    """Implement the notification entity for the Ecobee thermostat."""
+
+    _attr_name = None
+    _attr_has_entity_name = True
+
+    def __init__(
+        self, data: EcobeeData, thermostat_index: int, thermostat: dict
+    ) -> None:
+        """Initialize the thermostat."""
+        self.data = data
+        self.thermostat_index = thermostat_index
+        self.thermostat = thermostat
+        self._attr_unique_id = thermostat["identifier"]
+        self.update_without_throttle = False
```

Comment:
Both `self.thermostat` and `self.update_without_throttle` aren't used. Is it overwriting something I am missing or is it obsolete?